Make sure we don't try to start a session that's already started.

Avoids warnings like this in PHP 7.2:

PHP Warning:  session_set_cookie_params(): Cannot change session cookie parameters when session is active in [...]/functions/global.php on line 476

diff -Naurp a/functions/global.php b/functions/global.php
--- a/functions/global.php	2020-03-24 19:08:40.000000000 +0100
+++ b/functions/global.php	2020-06-20 10:34:06.928896213 +0200
@@ -455,7 +455,8 @@ function sqsession_destroy() {
  */
 
 function sqsession_is_active() {
-    sqsession_start();
+    if (session_status() === PHP_SESSION_NONE)
+        sqsession_start();
 }
 
 /**
